client/src/pages/Login/useStyles.ts

```diff
@@ -14,6 +14,9 @@ const useStyles = makeStyles(() => ({
     flexDirection: 'column',
     minHeight: '100vh',
     paddingTop: 23,
+    '& .makeStyles-authHeader-6': {
+      justifyContent: 'center',
+    },
```

Comment:
Could we just add direct styling onto the auth header as opposed to doing class selectors here?

Reply:
added breakpoints in authHeader to achieve the same functionality